Sources/SwiftDoc/Module.swift

```diff
@@ -22,27 +22,31 @@ public final class Module {
         let fileManager = FileManager.default
         for path in paths {
             let directory = URL(fileURLWithPath: path)
-            guard let directoryEnumerator = fileManager.enumerator(at: directory, includingPropertiesForKeys: [.isDirectoryKey], options: [.skipsHiddenFiles, .skipsPackageDescendants]) else { continue }
+            guard let directoryEnumerator = fileManager.enumerator(at: directory,
+                                                                   includingPropertiesForKeys: [.isDirectoryKey],
+                                                                   options: [.skipsHiddenFiles, .skipsPackageDescendants])
+            else { continue }
+
+            let ignoredDirectories: Set<URL> = Set([
+                "node_modules",
+                "Packages",
+                "Pods",
+                "Resources",
+                "Tests"
+            ].map { directory.appendingPathComponent($0) })
```

Comment:
It's also an option to use`.map(directory.appendingPathComponent))`

Reply:
Indeed! I considered that, but decided that cut against "making things clearer". Then again, are anonymous closure arguments (`$0`) _really_ better? 🙃